### DISCUSSION TITLE
amend commit message

### DISCUSSION THREAD

**Discussion by lnxfsf** [jesseduffield/lazygit#3044] [Q&A] [answered]
Is there option to, amend commit message  ?

I need to first have files in stash, before amending any commit. But what if i just want to change message of commit, or message or commit in history ?

**Comment by mark2185** [comment, marked as answer]
You can go to the `Commits` panel and press `r` on a commit, I believe.

Is that what you had in mind?

**Reply by CamiloTello002** [reply]
Thanks!!! I hadn't seen it xD

**Comment by bernd-wechner** [comment]
Just found this and like it. Have yet to test it, but I want to ask quickly before I get around to that, how this interacts with remotes? If I reword a commit and push to a remote is reworded there too?

**Reply by mark2185** [reply]
It works the same as with ordinary `git`, if the remote differs from local changes, you have to force push (`P`).
